privacy_policy.html

```diff
@@ -0,0 +1,236 @@
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Chapitô - Politique de confidentialité</title>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link rel="icon" type="image/png" href="img/favicon.png">
+
+    <!-- Lien pour la typographie-->
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i"
+          rel="stylesheet">
+
+</head>
+<body>
+<h1>Politique de Confidentialité</h1>
+
+
+<p>Date de prise d'effet: September 24, 2018</p>
+
+
+<p>Code-Troopers ("nous", "notre", "nos") exploite le site web et l'application mobile Chapitô (ci-après désignés par le
+    terme "Service").</p>
+
+<p>Cette page vous explique nos politiques en matière de collecte, d'utilisation et de communication des données à
+    caractère personnel lorsque vous utilisez notre Service ainsi que les choix qui s'offrent à vous en ce qui concerne
+    ces données. <a href="https://privacypolicies.com/free-privacy-policy-generator/">Politique de Confidentialité via
```

Comment:
Did you left that link purposefully ? If so, ignore me

Reply:
yes